src/remarkable-ui/components/editors/selects/shared/SelectFieldContent/SelectFieldContent.module.css

```
@@ -0,0 +1,35 @@
+.selectFieldContent {
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  box-sizing: border-box;
+
+  max-width: max(
+    var(--radix-dropdown-menu-trigger-width),
+    var(--em-selectfield-content-max-width, 25rem)
+  );
+  min-width: var(--radix-dropdown-menu-trigger-width);
+  max-width: var(--em-selectfield-content-max-width, 25rem);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

max-width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)